.storybook/storybook-data.js

```diff
@@ -65,7 +65,7 @@ module.exports = [
 			options: {
 				hierarchyRootSeparator: '|',
 				hierarchySeparator: {},
-				readySelector: '.googlesitekit-chart-v2 > div[style="position: relative;"]',
+				readySelector: '.googlesitekit-chart-v2',
```

Comment:
I think the intention here is to wait for the _inner_ selector – the one generated by a rendered chart, whereas `googlesitekit-chart-v2` will be present on all renderings of the chart, (loading, etc) so this may cause problems with the reliability of the test.

Reply:
Ah yes you are right, my bad. I've addressed this (hopefully!)